sources/packages/backend/apps/db-migrations/src/sql/ProgramYear/Create-program-year-2025-2026.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO
+    sims.program_years(
+        program_year,
+        program_year_desc,
+        form_name,
+        is_active,
+        start_date,
+        end_date,
+        parent_form_name,
+        partner_form_name,
+        program_year_prefix,
+        max_lifetime_bc_loan_amount
+    )
+VALUES
+    (
+        '2025-2026',
+        'Study starting between August 01, 2025 and July 31, 2026',
```

Comment:
Currently the program year descriptions are like this. 
![image](https://github.com/user-attachments/assets/9c76c301-7cb7-495b-9ada-effa266dd94c)

Is there a requirement to change the description into new format? let me know if I am missing something here.

Reply:
I just added the description (similar to the pre-existing ones as shown in your comment) for the newly added program year.

Reply:
Thanks @dheepak-aot . I had made the change you are referring to but just realized that I forgot to push the file to remote.